### PR TITLE
Fix code style with eslint

### PR DESCRIPTION
**Pull request by YurySolovyov**
Fix trivial issues reported by eslint.
For not-so-trivial, left comments like:

```
// TODO (eslint) ...
```

Refs #103

### CODE REVIEW

File: lib/bus.js [hunk 1]
```diff
@@ -1,356 +1,368 @@
...
+      errorName: errorName,
+      signature: 's',
+      body: [errorText]
+    };
+        // console.log('SEND ERROR', reply);
+    this.connection.message(reply);
+  };
+
+  this.sendReply = function (msg, signature, body) {
+    // TODO (eslint): use reply?
```
Comment (sidorares) [inline lib/bus.js]: oh, this was actually meant to be "reply to this call with with body/signature"  helper

could you add commit to fix to be `mesage(reply)` ?

File: lib/bus.js [hunk 2]
```diff
@@ -1,356 +1,368 @@
...
-       this.invokeDbus({ 'member': 'ListNames' }, callback);
-    };
+  this.listNames = function (callback) {
+    this.invokeDbus({'member': 'ListNames'}, callback);
+  };
 
-    this.listActivatableNames = function(callback) {
-       this.invokeDbus({ 'member': 'ListActivatableNames', signature: 's', body: [name]}, callback);
-    };
+  this.listActivatableNames = function (callback) {
```
Comment (sidorares) [inline lib/bus.js]: Looks like wrong signature (copy/paste error?). No arguments: https://dbus.freedesktop.org/doc/dbus-specification.html#bus-messages-list-activatable-names

Comment (YurySolovyov) [inline lib/bus.js]: not much help from me here :), just tell me how it should look like

File: lib/stdifaces.js [hunk 3]
```diff
@@ -7,198 +7,198 @@ var xmlHeader = '<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introsp
...
-        type: constants.messageType.error,
-        replySerial: msg.serial,
-        destination: msg.sender,
-        errorName: name,
-        signature: 's',
-        body: [ text ]
-    }
-    bus.connection.message(msg);
+function respondError (bus, msg, name, text) {
+  // TODO (eslint) use reply ?
```
Comment (sidorares) [inline lib/stdifaces.js]: same as for `sendReply`

Comment (YurySolovyov) [inline lib/stdifaces.js]: `respondError` itself is never used too.

**Comment by sidorares** [comment]
👍  thanks a lot, great work

I'll try to fix `TODO (eslint)`s in separate PR

**Comment by YurySolovyov** [comment]
@sidorares you can do a PR against my branch so we can merge entire work as one chunk.

**Comment by YurySolovyov** [comment]
@sidorares any news here?

**Comment by sidorares** [comment]
sorry @YurySolovyov completely forgot about this. Will look today

**Comment by jrobeson** [comment]
After looking through the other eslint PR, I kinda wanna adopt  `"eqeqeq": ["error", "always"], ` too. In a separate PR though.

**Comment by jrobeson** [comment]
@YurySolovyov : we did this with prettier instead, so we don't need the formatting options.

@sidorares: I kinda like the idea of adding eqeqeq though. Is there anything else you want?

**Comment by YurySolovyov** [comment]
I'm not sure this is relevant anymore

**Comment by jrobeson** [comment]
I did add the eqeqeq rule in #183, so I will close this now.

**Pull request closed without merging**